app/src/main/java/org/mozilla/tv/firefox/components/locale/LocaleAwareFragment.java

```diff
@@ -2,13 +2,12 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-
 package org.mozilla.tv.firefox.components.locale;
 
-import androidx.fragment.app.Fragment;
-
 import java.util.Locale;
 
+import androidx.fragment.app.Fragment;
+
 public abstract class LocaleAwareFragment extends Fragment {
```

Comment:
Nit: not sure why there's spacing between these imports.  Any chance you could run `optimize import statements` on this file before merging?

Reply:
Weird, optimize imports didn't do anything, but I removed that extra line anyway.